### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by hshahwan**
Create the footer component and makeup by writing CSS.

### CODE REVIEW

**Comment by sultanassi95** [comment]
### Is that ready to be merged? :thinking:

**Comment by sultanassi95** [comment]
Please, resolve conflicts and assign me as a reviewer when you are ready. @hshahwan

File: client/src/Components/Pages/HomePage/HeroSection/Style.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (thesoorae) [inline client/src/Components/Pages/HomePage/HeroSection/Style.css]: Make your spacing consistent for all css files. it looks like footer/style.css uses 4 spaces and this file uses 2.

Comment (sultanassi95) [reply client/src/Components/Pages/HomePage/HeroSection/Style.css]: Hope you guys make sure that u use the same Writing Style all over your projects, don't get frustrated by this, I believe that these soft skills are what makes you stand in-front when you are compared with other developers.

Comment (sultanassi95) [reply client/src/Components/Pages/HomePage/HeroSection/Style.css]: @RamyAlshurafa @MohammedYehia @hshahwan @marwajomaa

File: client/public/index.html [hunk 2]
```diff
@@ -15,8 +16,3 @@
   <body>
     <noscript>
 
-    </noscript>
-    <div id="root"></div>
-
-  </body>
-</html>
```
Comment (sultanassi95) [inline client/public/index.html]: why did u delete these lines!
I suppose that React app won't find the `id='root'` this way, and anyways, `html` and `body` tags most have a closing tag.

**Pull request merged**